packages/dapp/src/components/MultisigConfirmationModal/MultisigConfirmationModal.tsx

```diff
@@ -30,10 +30,10 @@ export const MultisigConfirmationModal: React.FC<{
             fontSize: '1.125rem',
           }}
         >
-          {translate(`${i18Path}.title`)}
+          {translate(`${I18_PATH}title`, 'translation')}
```

Comment:
@Addminus  @chybisov Also, please take note on this usage of the translate-hook. Been using this now all over the place to fix the warnings on TemplateStringArrays. This seems to do the trick but I´d like the have you check since that´s quite some changes across all of our files.

Reply:
Also, I'm not sure why we started to use i18n like this

```ts
const { t: translate } = useTranslation();
```
but it would be cool to have it as default `t` everywhere. Fewer letters to type, and looks compact. 🙂

```ts
const { t } = useTranslation();
```